modules/i18n/src/main/I18nKeys.scala

```diff
@@ -106,6 +106,7 @@ final class I18nKeys(translator: Translator) {
   val `players` = new Key("players")
   val `minutesPerSide` = new Key("minutesPerSide")
   val `variant` = new Key("variant")
+  val `variants` = new Key("variants")
```

Comment:
this file should never be edited by hand, but generated with `bin/gen/trans-dump`